sample-dcrdata.conf

```diff
@@ -42,6 +42,10 @@
 ; syncing is done.
 ; sync-status-limit=1000
 
+; Blocks logging of the PostgreSQL db configuration on system start up. Defaults
```

Comment:
Just "Block logging of the PostgreSQL db configuration on system start up." Default for booleans are an implied false when the switch is omited.